rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/message/MessageFormLayout.java

```diff
@@ -159,23 +151,11 @@ public void run() {
     });
   }
 
-  private void showExtraActionSelectionDialog() {
-    ExtraActionPickerDialog.showAsTask(getContext(), extraActionItems)
-        .onSuccess(new Continuation<Integer, Object>() {
-          @Override
-          public Object then(Task<Integer> task) throws Exception {
-            int which = task.getResult();
-            if (actionListener != null) {
-              actionListener.onExtraActionSelected(which);
-            }
-            return null;
-          }
-        });
-  }
-
-  public interface ActionListener {
-    void onSubmitText(String message);
+  public interface ShowExtraActionSelectionCallback {
+    void show();
+  }
 
-    void onExtraActionSelected(int itemId);
+  public interface TextListener {
+    void onSubmitText(String message);
```

Comment:
I think it is better to include the word `Submit` in the name of interface, like `SubmitTextCallback`